test/e2e/functional/tests/e2e.go

```diff
@@ -837,8 +840,9 @@ var _ = Describe("Ingress Node Firewall", func() {
 			Expect(err).ShouldNot(HaveOccurred())
 			//wait for daemonset to be rolled out
 			infDaemonSet := &appsv1.DaemonSet{}
-			err = daemonset.WaitForDaemonSetReady(testclient.Client, infDaemonSet, OperatorNameSpace,
-				"ingress-node-firewall-daemon", retryInterval, timeout)
+			infDaemonSet.SetName("ingress-node-firewall-daemon")
```

Comment:
Nit: pls use https://github.com/openshift/ingress-node-firewall/blob/master/test/consts/const.go#L15 for the ds name